src/servers/model/server/api.ts

```diff
@@ -288,7 +291,8 @@ export function normalizeRestCommonParams(params: any): CommonQueryParamsInfo {
         model_nums: params.model_nums ? ('' + params.model_nums).split(',').map(n => n.trim()).filter(n => !!n).map(n => +n) : void 0,
         data_source: params.data_source,
         copy_all_categories: Boolean(params.copy_all_categories),
-        encoding: mapEncoding(('' + params.encoding).toLocaleLowerCase())
+        encoding: mapEncoding(('' + params.encoding).toLocaleLowerCase()),
+        transform: params.transform ? createMat4(('' + params.transform).split(',').map(n => n.trim()).map(n => +n)) : Mat4.identity()
```

Comment:
You can just cast `as Mat4`, `Mat4` is just a number array of length 16.

Why convert to a string and parse it again?

Reply:
I followed the example of `model_nums` which will also create a number[]. `typeof params.transform` reports string at the start of the method. I understood the explicit conversion to string as safety net (though unsure against which scenario).

`as Mat4` will compile but fail at runtime because the type of matrix elements is still string.

Reply:
He meant that instead of calling ``createMat4`` (which can be removed) you can just do ``as Mat4``. Parsing the numbers is still required.

Reply:
Oh, sure thing. Done in 6f05179.